app/src/main/java/org/fossasia/openevent/api/processor/MicrolocationListResponseProcessor.java

```diff
@@ -33,10 +37,14 @@ public void success(MicrolocationResponseList microlocationResponseList, Respons
 
         dbSingleton.clearDatabase(DbContract.Microlocation.TABLE_NAME);
         dbSingleton.insertQueries(queries);
+
+        Bus bus = OpenEventApp.getEventBus();
+        bus.post(new MicrolocationDownloadEvent(true));
```

Comment:
Same here? Why isn't the query inserting being done in a worker thread, i thought we decided to use that